packages/sdk-core/src/modules/rateLimiter/RateLimitWatcher.ts

```diff
@@ -0,0 +1,60 @@
+import { TimeHelper } from '../../common/TimeHelper';
+
+export class RateLimitWatcher {
+    /**
+     * Time the single report can stay in the queue.
+     */
+    public MAXIMUM_TIME_IN_QUEUE = 60;
```

Comment:
Can we use milliseconds here?

Reply:
our reports use timestamps in sec required by Backtrace to send successful reports. I prefer to stay to this instead of adding ms here.